dietpi/dietpi-cloudshell

```diff
@@ -1492,6 +1498,18 @@ _EOF_
 		do
 
 			aEnabledScenes[$i]=1
+			# Configure Tor Relay
+			if [[ $i == 9 ]]; then
+				G_AGI nyx
```

Comment:
Use `G_AG_CHECK_INSTALL_PREREQ nyx` here, which does not always call `apt-get` but only if the package is not yet installed. So after having this selected the first time, it will be done much quicker.